Integrations/DomainTools_Iris/iris.py

```diff
@@ -0,0 +1,393 @@
+            "NameServers": name_servers,
+            "SSLCertificate": ssl_certificates,
+            "RedirectsTo": redirects_to,
+            "GoogleAdsenseTrackingCode": google_adsense,
+            "GoogleAnalyticTrackingCode": google_analytics
+        }
+    }
+
+    domain_context = {
+        "Name": domain,
+        "DNS": ip_addresses,
+        "Vendor": "DomainTools",
+        "CreationDate": create_date,
+        "RiskScore": overall_risk_score,
+        "DomainStatus": domain_status,
+        "ExpirationDate": expiration_date,
+        "NameServers": domain_name_servers,
+        "Registrant": domain_registrant_contact,
+        "DomainTools": domain_tools_context
+    }
+    if overall_risk_score and overall_risk_score >= RISK_THRESHOLD:
+        domain_context["Malicious"] = {
+            "Vendor": 'DomainTools',
+            "Description": threat_profile_evidence
+        }
+
+    return domain_context
+
+
+def format_domain_profile(response):
+    domain_result = response['results'][0]
+    domain_context = create_context_output(domain_result)
+    outputs = {"Domain(val.Name == obj.Name)": domain_context}
+    convert_empty_to_null(outputs)
+
+    human_readable = '### DomainTools Profiled {}. See context data for more info.'.format(domain_result['domain'])
+    return human_readable, outputs
+
+
+def format_domain_analytics(response):
+    domain_result = response['results'][0]
+    overall_risk_score = domain_result['domain_risk']['risk_score']
+    # The index 0 refers to "proximity" or "blacklist"
+    proximity_risk_score = domain_result['domain_risk']['components'][0]['risk_score']
+    domain_age = find_age(domain_result['create_date']['value'])
+    website_response = domain_result['website_response']
+    google_adsense = domain_result['adsense']['value']
+    google_analytics = domain_result['google_analytics']['value']
+    alexa_rank = domain_result['alexa']
+    tags = ', '.join(domain_result['tags'])
+
+    human_readable_data = {
+        "Overall Risk Score": overall_risk_score,
+        "Proximity Risk Score": proximity_risk_score,
+        "Domain Age (in days)": domain_age,
+        "Website Response": website_response,
+        "Google Adsense": google_adsense,
+        "Google Analytics": google_analytics,
+        "Alexa Rank": alexa_rank,
+        "Tags": tags
+    }
+
+    convert_empty_to_null(human_readable_data)
+
+    outputs = {}  # type: Dict[Any, Any]
+
+    headers = ["Overall Risk Score",
+               "Proximity Risk Score",
+               "Domain Age (in days)",
+               "Website Response",
+               "Google Adsense",
+               "Google Analytics",
+               "Alexa Rank",
+               "Tags"]
+    human_readable = tableToMarkdown('DomainTools Domain Analytics for {}.'.format(domain_result['domain']),
+                                     human_readable_data,
+                                     headers=headers)
+    return human_readable, outputs
```

Comment:
You are always returning empty dictionary here in `outputs`, what are you trying to do ?